x-pack/plugins/timelines/common/ecs/rule/index.ts

```diff
@@ -8,6 +8,7 @@
 export interface RuleEcs {
   id?: string[];
```

Comment:
`id` is gone, unless we need it here for legacy signals?

Reply:
We shouldn't with your mapping. 👍🏾